src/aim/utils/performance_test.py

```
@@ -0,0 +1,413 @@
+            max_concurrent_users=50,
+            test_duration=60
+        )
+        
+        print("\nStress Test Results:")
+        for users, results in stress_results.items():
+            print(f"  {users} users: {results.requests_per_second:.2f} req/s, "
+                  f"{results.error_rate:.1f}% errors")
+        
+        # Generate comprehensive report
+        print("\n4. Generating Report")
+        report = tester.generate_report(health_results, "performance_report.txt")
```

Ruff (0.12.2)

Local variable `report` is assigned to but never used

Remove assignment to unused variable `report`

(F841)